ROADMAP.md

```diff
@@ -0,0 +1,33 @@
+# Kubeflow Distributed Training Operators 2020 Roadmap
+
+This document outlines the main directions on the Kubeflow Distributed Training Operators in 2020.
+
+## Maintenance and reliability
+
+We will continue developing capabilities for better reliability, scaling, and maintenance of production distributed training experiences provided by operators.
+
+* Enhance maintainability of operator common module https://github.com/kubeflow/common/issues/54
+* Migrate operators to use kubeflow/common apis
+* Graduate MPI Operator, Mxnet Operator and XGBoost Operator to v1
+
+## Features
+
+To take advatanges of other capabilities of job scheduler components, operators will expose more APIs for advanced scheduling. More features will be added to simplify usage like dynamic volume supports and git ops experiences. In order to make it easily used in the Kubeflow ecosystem, we can add more launcher KFP components for adoption.
```

Comment:
From our customers, users have different options for training component in KFP. We have cloud components like Sagemaker, lots of users want to use Kubeflow based training operator as well. In order to use TF/MPI smoothly, it would be better to write some common launcher KFP component like https://github.com/kubeflow/pipelines/tree/master/components/kubeflow/katib-launcher

User just need to pass count of PS, Worker, Container Image, argument ,etc to kind of the job and output artifacts. This is much easier compare to writing a yaml and use [ResouceOp](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.dsl.html#kfp.dsl.ResourceOp)

My point is we need high level abstraction of job to increase operator adoption in KF ecosystem

Reply:
Does tf launcher https://github.com/kubeflow/pipelines/tree/master/components/kubeflow/launcher here align to what you describe here?
Then what we need to do next is to introduce other Jobs launcher or provide a common launcher.

Reply:
Yes. We plan to have a common launcher. Using operator separately will be hard to virtual artifacts, model metrics. We have to use KFP in this case or create separate component to integrate these features for training jobs